src/include/sof/lib/memory_pm.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Shouldn't this be a part of pm_runtime?

Reply:
Not sure i follow You here - Can You be more specific ?

Reply:
The whole memory power management functionality shall be the part of pm_runtime API.

Reply:
The pm_runtime will be also in this pr as soon as the memory pm api works i will add relative calls and structs in pm_runtime as a separate commit.

Reply:
That differs from what was expected of me in PRs before where usage of an api and api itself had to be in separate commits.